lib/services/screen-builder-service.ts

```diff
@@ -131,17 +131,35 @@ export class ScreenBuilderService implements IScreenBuilderService {
 $injector.register("screenBuilderService", ScreenBuilderService);
 
 class ScreenBuilderDynamicCommand implements ICommand {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
```

Comment:
It should be just `.app.json`. That's what is used on the server to check whether or not a given project is ScreenBuilder compatible.